isce2_topsapp/localize_mask.py

```diff
@@ -19,17 +19,27 @@ def download_water_mask(
         np.ceil(extent_buffered[3]),
     ]
 
-    if water_name == "SWDB":
+    if water_name == "SWBD":
         # Download SRTM-SWDB water mask
-        mask_filename = download_wbd(
-            extent_buffered[1],
-            extent_buffered[3],
-            extent_buffered[0],
-            extent_buffered[2],
-        )
+        # Water mask dataset extent
+        # Latitude S55 - N60
+
+        lats = [extent_buffered[1], extent_buffered[3]]
+        if (np.abs(lats) < 59.9).all():
```

Comment:
Just to clarify, the lats > -56 (not sure if the remaining 3 degrees in Southern hemisphere is just water)

```suggestion
        if ((lats < 59.9).all()) and ((lats > -55.9).all()):
```

Reply:
SWBD coverage goes up to S54 deg, below data is only water so do not think we are going to do any processing there

it was just easier to write it like this using abs.